packages/core/src/external/aws/lambda-logic/bundle-to-html.ts

```diff
@@ -586,25 +590,102 @@ function createDiagnosticReportsSection(
     false
   );
 
-  const hasNonAWEreports = nonAWEreports.length > 0;
-
+  const reportsFromB64 = createDiagnosticReportsWithB64Notes(diagnosticReports, observations);
+  const noReportsPlaceholder = `<table><tbody><tr><td>No reports found</td></tr></tbody></table>`;
   return `
     <div id="reports" class="section">
       <div class="section-title">
         <h3 id="reports" title="reports">&#x276F; Reports</h3>
         <a href="#mr-header">&#x25B2; Back to Top</a>
       </div>
       <div class="section-content">
-        ${
-          hasNonAWEreports
-            ? nonAWEreports
-            : `<table><tbody><tr><td>No reports found</td></tr></tbody></table>`
-        }
+        ${combineStrings(nonAWEreports, reportsFromB64, noReportsPlaceholder)}
       </div>
     </div>
   `;
 }
 
+function combineStrings(
+  stringA: string | undefined,
+  stringB: string | undefined,
+  placeholder = "default"
```

Comment:
nit: the presence of a default here makes the function name confusing. I would have this return undefined if the two string are undefined, and then based on that logic use the noReportsPlaceholder in the html

Reply:
woops, this is a leftover